app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -23,6 +24,10 @@ public DBHelper(Context context) {
         open();
     }
 
+    /**
+     * Inserts museum data to museums table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.